### PR TITLE
Hunters cart

### PR DESCRIPTION
**Pull request by Hunter-aw**
I added in cart Toggle function (to toggle the cart)
add cart (add items to my cart array), 
update cart (render) via handlebars
cart total (calculate total)
and clear cart (clear array and reset total)

### CODE REVIEW

File: public/script.js [hunk 1]
```diff
@@ -1,28 +1,57 @@
...
+      price: price
+    }
+    cart.push(newItem);
+    
     // TODO: Write this function. Remember this function has nothing to do with display. 
     // It simply is for adding an item to the cart array, no HTML involved - honest ;-)
   }
+  var cartTotal = function () {
+    var total = 0
+    for (i in cart) {
```
Comment (JonaCodes) [inline public/script.js]: let

Comment (JonaCodes) [reply public/script.js]: better yet, use `of`, then you don't need to do `cart[i]`

File: public/script.js [hunk 2]
```diff
@@ -1,28 +1,57 @@
...
+      let price = cart[i].price;
+      total += price
+    }
+    $('.total').html(total)
```
Comment (JonaCodes) [inline public/script.js]: why not handlebars?

File: public/script.js [hunk 3]
```diff
@@ -35,15 +64,19 @@ app.updateCart();
...
 $('.view-cart').on('click', function () {
-  // TODO: hide/show the shopping cart!
-});
+    $('.shopping-cart').toggle()
+  });
 
 $('.add-to-cart').on('click', function () {
   // TODO: get the "item" object from the page
-  app.addItem(item);
+  var $item = $(this).closest('.item')
```
Comment (JonaCodes) [inline public/script.js]: const

File: server.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const express = require('express');
+let app = express()
+app.listen(8008)
```
Comment (JonaCodes) [inline server.js]: good practice to have the listen at the end of the file, and indicate that it's running with a log just before

File: server.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+app.use(express.static('public'));
+app.use(express.static('node_modules'));
```
Comment (JonaCodes) [inline server.js]: use `__dirname + '/public'` (and same for node_modules) - this guarantees our path is what we want it to be (important when we'll actually host our websites off our computer

**Review by JonaCodes — CHANGES_REQUESTED** [review]

File: public/script.js (hunk 1 above)
Comment (Hunter-aw) [reply public/script.js]: that makes sense :P

File: public/script.js (hunk 2 above)
Comment (Hunter-aw) [reply public/script.js]: I didn't feel it necessary for a single line, and it's not in my object.

**Pull request merged**